--- a/modules/videoio/cmake/detect_ffmpeg.cmake
+++ b/modules/videoio/cmake/detect_ffmpeg.cmake
@@ -4,7 +4,7 @@
     set(OPENCV_FFMPEG_USE_FIND_PACKAGE "FFMPEG")
   endif()
   find_package(${OPENCV_FFMPEG_USE_FIND_PACKAGE}) # Required components: AVCODEC AVFORMAT AVUTIL SWSCALE
-  if(FFMPEG_FOUND OR FFmpeg_FOUND)
+  if(FFMPEG_FOUND OR FFmpeg_FOUND OR ffmpeg_FOUND)
     set(HAVE_FFMPEG TRUE)
   endif()
 endif()
